eva_submission/nextflow/remap_and_cluster.nf

```diff
@@ -253,3 +254,33 @@ process qc_clustering {
         > ${params.target_assembly_accession}_clustering_qc.log
     """
 }
+
+
+/*
+ * Run Back propagation of new clustered RS
+ */
+process backpropagate_clusters {
+    memory "${params.memory}GB"
+    clusterOptions "-g /accession"
+
+    when:
+    source_assembly_accession != params.target_assembly_accession
```

Comment:
Is this `when` necessary?  This should only run if there's both a remapped VCF and a clustering QC log, which I think is sufficient.

Reply:
True. I removed it.